packages/rocketchat-ui-flextab/client/tabs/userInfo.html

```diff
@@ -60,16 +60,10 @@ <h3 title="{{name}}" class="rc-user-info__name"><i class="status-{{status}}"></i
 							{{#if hasPermission 'view-full-other-user-info'}}
 								{{#if hasEmails}}
 									<div class="rc-user-info-details__item">
-										<label class="rc-user-info-details__label	">Email address</label>
+										<label class="rc-user-info-details__label	">{{_ "Email"}}</label>
 										{{#each emails}} <a href="mailto:{{address}}" class="rc-user-info-details__info">{{address}}{{#if verified}}&nbsp;<i class="icon-ok success-color"></i>{{/if}}</a> {{/each}}
 									</div>
 								{{/if}}
-								{{#if hasPhone}}
```

Comment:
Why did you remove the phone field?

Reply:
I brought it back, but I'm not sure how to add it as a custom field to test it.

Reply:
It's a livechat thing